### ISSUE TITLE
🚧Setup dependabot

### ISSUE THREAD

**Issue by CalvinWilkinson** [KinsonDigital/GotNuget#29]
### Complete The Item Below

- [X] I have updated the title without removing the 🚧 emoji.

### Description

Setup [dependabot](https://docs.github.com/en/code-security/dependabot) for the project.

Refer to the **Velaptor** project for the how to setup the **YAML** config tile.

The name of the file is [dependabot.yml](https://github.com/KinsonDigital/Velaptor/blob/e228cf09e27bdc45c23806ed0e80a54c94f3ec76/.github/dependabot.yml#L4)

### Acceptance Criteria

- [x] Dependabot setup for project

### ToDo Items

- [X] Change type labels added to this issue.  Refer to the _**Change Type Labels**_ section below.
- [X] Priority label added to this issue.  Refer to the _**Priority Type Labels**_ section below.
- [X] Issue linked to the correct project _(if applicable)_.
- [x] Issue linked to the correct milestone _(if applicable)_.
- [x] Draft pull request created and linked to this issue _(only required with code changes)_.

### Issue Dependencies

- #32 
    > **Note** This is because if this issue is completed first, it will start the bot and all of the NuGet package updates will be detected and a lot of PR's will be created.  The initial update of the NuGet packages should be done first to help save work.

### Additional Information:

**_<details closed><summary>Change Type Labels</summary>_**

| Change Type         | Label                |
|---------------------|----------------------|
| Bug Fixes           | `🐛bug` |
| Breaking Changes    | `🧨breaking changes` |
| New Feature         | `✨new feature` |
| Workflow Changes    | `workflow` |
| Code Doc Changes    | `🗒️documentation/code` |
| Product Doc Changes | `📝documentation/product` |
</details>

**_<details closed><summary>Priority Type Labels</summary>_**

| Priority Type       | Label             |
|---------------------|-------------------|
| Low Priority        | `low priority`    |
| Medium Priority     | `medium priority` |
| High Priority       | `high priority`   |
</details>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.
